src/Enum.php

```diff
@@ -130,15 +139,11 @@ protected static function resolve(): array
             $enumValues[$value] = $name;
         }
 
-        self::$cache[$class] = $enumValues;
-
-        return self::$cache[$class];
+        return self::$cache[$class] = $enumValues;
```

Comment:
I find that this shorthand notation makes the code less clear to read. I'd prefer you'd change it back to the original.